core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.ts

```diff
@@ -468,6 +480,18 @@ angular.module('oppia').component('settingsTab', {
           'vertical-align': 'top'
         });
       };
+
+      ctrl.refreshPermissions = function() {
```

Comment:
is this function used in the template? If not, then can we avoid attaching it to ctrl?

Reply:
DONE